moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -459,107 +614,6 @@ std::vector<OMPLPlannerDescription> MoveItConfigData::getOMPLPlanners()
   return planner_des;
 }
 
-// ******************************************************************************************
```

Comment:
just curious - why move these files? it makes reviewing harder because i can't tell what has actually changed

Reply:
Sorry I didn't understand, which files?

Reply:
oops i mean functions.... why move the functions to a different location in the file?